spec/controllers/groups_controller_spec.rb

```diff
@@ -0,0 +1,121 @@
+require 'rails_helper'
+
+RSpec.describe GroupsController, type: :controller do
+  let(:group) { create(:group) }
+  let(:valid_attr) { attributes_for(:group) }
+  let(:invalid_attr) { attributes_for(:group, title: '') }
+  let(:new_attr) { attributes_for(:group, title: 'Blåtand') }
+
+  allow_user_to :manage, Group
+
+  describe "GET #index" do
+    it "assigns" do
+      group
+      get :index
+      assigns(:groups).should include(group)
+    end
+  end
+
+  describe "GET #show" do
+    it "assigns the requested group as @group" do
+      get :show, id: group.to_param
+      assigns(:group).should eq(group)
+    end
+  end
+
+  describe "GET #new" do
+    it "assigns a new group as @group" do
+      get :new
+      assigns(:group).should be_a_new(Group)
+    end
+  end
+
+  describe "GET #edit" do
+    it "assigns the requested group as @group" do
+      get :edit, id: group.to_param
+      assigns(:group).should eq(group)
+    end
+  end
+
+  describe "POST #create" do
+    context "with valid params" do
+      it "creates a new Group" do
+        lambda do
+          post :create, group: valid_attr
+        end.should change(Group, :count).by(1)
+      end
+
+      it "assigns a newly created group as @group" do
+        post :create, group: valid_attr
+        assigns(:group).should be_a(Group)
+        assigns(:group).should be_persisted
+      end
+
+      it "redirects to the created group" do
+        post :create, group: valid_attr
+        response.should redirect_to(Group.last)
+      end
+    end
+
+    context "with invalid params" do
+      it "assigns a newly created but unsaved group as @group" do
+        post :create, group: invalid_attr
+        assigns(:group).should be_a_new(Group)
+      end
+
+      it "re-renders the 'new' template" do
+        post :create, group: invalid_attr
+        response.should render_template("new")
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.